src/playground/with-layout/chat/chat-custom-message.component.scss

```
@@ -0,0 +1,12 @@
+nb-chat {
+  width: 500px;
```

```suggestion
  width: 500px;
  margin: 0 auto;
```
Add margin to center chat component horizontally.